tests/test_servicemesh.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -xeEo pipefail
+
+source tests/common.sh
+
+function finish {
+  if [ $? -eq 1 ] && [ $ERRORED != "true" ]
+  then
+    error
+  fi
+
+  echo "Cleaning up ServiceMesh"
+  long_uuid=$(get_uuid 0)
+  uuid=${long_uuid:0:8}
+  kubectl delete --ignore-not-found=true namespace example-benchmark-controlplane-$uuid
+  kubectl delete --ignore-not-found=true namespace example-benchmark-workload-$uuid
+  kubectl delete -f tests/test_crs/valid_servicemesh.yaml
+  delete_operator
```

Comment:
can we sub in the wait_clean function instead to also clean up the namespace etc.